### PR TITLE
Create pr6

### PR DESCRIPTION
**Pull request by iluxa-c0m**

### CODE REVIEW

**Comment by iluxa-com** [comment]
IDE

**Comment by GH-integration** [comment]
gh?

File: pr6 [hunk 1]
```diff
@@ -0,0 +1,2 @@
+12
```
Comment (maxim5) [inline pr6]: gh +12

**Comment by maxim5** [comment]
|`pr6:1`|
|---|
up +12 reply

**Comment by maxim5** [comment]
|`pr6:1`|
|---|
up +12 reply

File: pr6 (hunk 1 above)
Comment (maxim5) [inline pr6]: up

File: pr6 [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+123
```
Comment (maxim5) [inline pr6]: up2

File: pr6 [hunk 3]
```diff
@@ -0,0 +1 @@
+12
```
Comment (maxim5) [inline pr6]: gh +12 in 79ecb3f315618a600bd114591852a3c72fa90601

File: pr6 (hunk 1 above)
Comment (maxim5) [inline pr6]: up +12

**Pull request closed without merging**